### PR TITLE
Vexination API and Indexer resources

### PR DESCRIPTION
**Pull request by mrrajan**
This PR includes,
 - Vexination API resources
 - README.md updates for variables
 - example variable files for AWS and Non-AWS services.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,20 +93,44 @@ Copy your certificate files in `./certs` directory using following names:
...
+export TPA_OIDC_ISSUER_URL=<AWS Cognito or Keycloak Issuer URL. Incase of Keycloak endpoint auth/realms/chicken is needed>
+export TPA_OIDC_FRONTEND_ID=<OIDC Frontend Id>
 export TPA_OIDC_PROVIDER_CLIENT_ID=<OIDC Walker Id>
 export TPA_OIDC_PROVIDER_CLIENT_SECRET=<OIDC Walker Secret>
 export TPA_EVENT_ACCESS_KEY_ID=<Kafka Username or AWS SQS Access Key>
 export TPA_EVENT_SECRET_ACCESS_KEY=<Kafka User Password or AWS SQS Secret Key>
-export TPA_EVENT_BOOTSTRAP_SERVER=<Kafka Bootstrap Server URL, For AWS SQS it is not needed>
 ```
 
+In case of Kafka Events, create environmental variable for bootstrap server
```
Comment (desmax74) [inline README.md]: Add a number on this event section like the others ,to be visually separated as argument and not aggregated to the Envs section

**Review by desmax74 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mrrajan) [reply README.md]: Updated :)

**Review by gildub — APPROVED** [review]
LGTM

File: roles/tpa_single_node/vars/main.yml [hunk 2]
```diff
@@ -12,52 +12,51 @@ tpa_single_node_systemd_directory: /etc/systemd/system
 tpa_single_node_default_empty: ""
 
 # DB_SERVICE
-tpa_single_node_pg_host: "{{ lookup('env', 'TPA_PG_HOST' default='192.168.121.60') }}"
+tpa_single_node_pg_host: 192.168.121.60
```
Comment (gildub) [inline roles/tpa_single_node/vars/main.yml]: The lookup is good idea meanwhile there is a typo.
```suggestion
tpa_single_node_pg_host: "{{ lookup('env', 'TPA_PG_HOST') | default('192.168.121.60', true) }}"
```

**Review by gildub — COMMENTED** [review]
Just wondering why you changed that TPA_PG_HOST back ?

**Review by desmax74 — APPROVED** [review]

**Pull request merged**